provers/celestia-prover/prover/Cargo.toml

```diff
@@ -2,6 +2,10 @@
 name = "celestia-prover"
 version = "0.1.0"
 edition = { workspace = true }
+members = [
+    "../programs/sp1/mock-membership",
+    "../programs/sp1/mock-update-client",
+]
```

Comment:
[question] what does including these programs as members of the celestia-prover crate do? They're already members of the root level workspace [here](https://github.com/celestiaorg/celestia-zkevm-ibc-demo/blob/776f44a8471422a8754e464c32a10034dffea376/Cargo.toml#L4-L5). I don't quite understand when/why to include them as members of this crate.

Reply:
I think if you want to share the packages that are defined in the parent package is one of the reasons. I don't fully understand either. I was trying to debug compiling the mock programs and was trying to see if any of these would help.